src/tools/gen_code/Makefile.am

```diff
@@ -9,6 +8,6 @@ gen_code_CPPFLAGS = -I../../ta_common
 gen_code_LDFLAGS = -L../../ta_common -L../../ta_abstract -L../../ta_func
 gen_code_LDADD = -lta_common -lta_abstract_gc -lta_func -lm
 
-all-local:
-	$(MAKE) $(AM_MAKEFLAGS) gen_code
-	cp gen_code ../../../bin
+# all-local:
```

Comment:
I'd prefer to leave it unchanged. Is it possible to not install the `gen_code` script in the `PKGBUILD`?

Reply:
This was triggering a recursive `make` that was screwing up the compilation when using parallel compilation.